server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -91,7 +92,10 @@ CURRENCY : C U R R E N C Y;
 DATE_COMPILED : D A T E MINUSCHAR C O M P I L E D;
 DATE_WRITTEN : D A T E MINUSCHAR W R I T T E N;
 DAY_OF_WEEK : D A Y MINUSCHAR O F MINUSCHAR W E E K;
+DB_KEY : D B MINUSCHAR K E Y;
 DBCS : D B C S;
+DBNAME : D B N A M E;
```

Comment:
Optional: I guess these two tokens can be moved to CommonCobolLexer (to re-usage), e.g. can be used as a host-variable